### ISSUE TITLE
Hello, how many layers of neural network are you using to extract features? Is the classifier softmax?Where is these content in the code?

### ISSUE THREAD

**Issue by muyishengfei** [xionghc/Facial-Expression-Recognition#35]
Hello, how many layers of neural network are you using to extract features? Is the classifier softmax?Where is these content in the code?

**Comment by xionghc** [comment]
The neural network model was defined in `model.py`. You can find the details of neural network from there. Recently, I intend to replace this model by Resnet18.
